src/ToDo.UI/Views/TaskNotePage.xaml

```diff
@@ -29,6 +66,35 @@
 				</AppBarButton>
 			</utu:NavigationBar.MainCommand>
 		</utu:NavigationBar>
+		<utu:AutoLayout grid.Row="2">
+		<utu:AutoLayout utu:AutoLayout.PrimaryAlignment="Stretch">
+			<uer:FeedView Source="{Binding Entity}" utu:AutoLayout.PrimaryAlignment="Stretch">
+				<DataTemplate>
+					<utu:AutoLayout Spacing="10" Padding="0,10,0,0" DataContext="{Binding Data}">
+						<TextBox PlaceholderText="Add Note" utu:AutoLayout.PrimaryAlignment="Stretch" Style="{StaticResource MaterialFilledTextBoxStyle}" x:Uid="Body_TextBox" />
+						<utu:AutoLayout Padding="20,0,10,0">
+							<utu:AutoLayout Spacing="16" Orientation="Horizontal">
+								<utu:AutoLayout utu:AutoLayout.CounterAlignment="Center" utu:AutoLayout.PrimaryAlignment="Stretch">
+									<utu:AutoLayout PrimaryAxisAlignment="Center">
+										<utu:AutoLayout PrimaryAxisAlignment="Center">
+											<TextBlock Foreground="{ThemeResource MaterialOnSurfaceMediumBrush}" TextWrapping="Wrap" Text="Auto save" Style="{StaticResource MaterialBody2}" x:Uid="Body_Subtitle" />
+										</utu:AutoLayout>
+									</utu:AutoLayout>
+								</utu:AutoLayout>
+								<utu:AutoLayout Orientation="Horizontal" utu:AutoLayout.CounterAlignment="Center">
+									<utu:AutoLayout PrimaryAxisAlignment="End" Orientation="Horizontal">
+										<Button utu:AutoLayout.CounterAlignment="Start" Style="{StaticResource MaterialButtonIconStyle}" x:Uid="Body_IconButton">
+											<PathIcon Data="{StaticResource Icon_Delete_Outline}" />
+										</Button>
+									</utu:AutoLayout>
+								</utu:AutoLayout>
+							</utu:AutoLayout>
+						</utu:AutoLayout>
+					</utu:AutoLayout>
+				</DataTemplate>
+			</uer:FeedView>
+		</utu:AutoLayout>
+	</utu:AutoLayout>
 
-	</Grid>
+	</Grid>-->
```

Comment:
Is this generated? Otherwise is it expected to comment a part of the page and add lot of commented code?

Reply:
I'm going to assume this is a dump of the generated code, so leaving it here